### PR TITLE
Partner page

### LINKED ISSUE #96: Partners
## Partners

Crediting Initiatives who are lined up to Build One Initiatives.

- Add Your Logos

### PR DESCRIPTION
**Pull request by peeyush14goyal**
closes #96 

- [x] Added Partner page in /partners
- [x] Current Partners link in about page partners section

### CODE REVIEW

File: components/PartnerCard.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const PartnerCard = ({ name, logoURL, url }) => {
+    return (
+        <div className="inline-block">
+            <a href={url} target="_blank">
+                <span
+                    className=" max-w-md shadow my-3 inline-block mx-6 border-2 border-gray-300
+        dark:border-gray-900 cursor-pointer hover:transform transition duration-300 ease-in-out hover:scale-105">
```
Comment (Deep-Codes) [inline components/PartnerCard.jsx:9]: Remove the Hover and add Small Border Radius

**Review by Deep-Codes — APPROVED** [review]

**Comment by Deep-Codes** [comment]
Just do Changes and ready to Merge.

File: components/PartnerCard.jsx (hunk 1 above)
Comment (peeyush14goyal) [reply components/PartnerCard.jsx]: Changes done.

Comment (Deep-Codes) [reply components/PartnerCard.jsx]: Merge it Then

**Pull request merged**